src/bin/nps.js

```diff
@@ -102,6 +102,24 @@ function onInit() {
   ))
 }
 
+function getConfigType() {
+  let configType
+
+  if (includes(process.argv, '-t')) {
```

Comment:
If we're going to allow `-t` then we should probably document that. Personally, I'd be good to not allow `-t` and just accept `--type`